### PR TITLE
Add stats logging

### PR DESCRIPTION
**Pull request by davisp**
Yay stats!

### CODE REVIEW

File: c_src/util.cc [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    exit(255);
+}
+
+void
+init_signals()
+{
+    signal(SIGINT, show_stack);
+    signal(SIGQUIT, show_stack);
+    signal(SIGABRT, show_stack);
+    signal(SIGKILL, show_stack);
```
Comment (nickva) [inline c_src/util.cc]: This probably can't be caught?

Comment (davisp) [reply c_src/util.cc]: Sure can't.

Comment (davisp) [reply c_src/util.cc]: Removed.

**Review by nickva — APPROVED** [review]
+1

**Pull request merged**